### DISCUSSION TITLE
Stereo sound for the Ant Media Server v2.7.0

### DISCUSSION THREAD

**Discussion by Mohit-3196** [ant-media/Ant-Media-Server#5794] [General]
Hello,
I recently checked the [release logs](https://github.com/ant-media/Ant-Media-Server/releases) for Ant Media Server version 2.7.0 and realized that there's support for high-quality Audio and Stereo audio as well for WebRTC publish and Play.

How can I test these improvements and are there any additional set of steps/configurations that I need to do in order to enable the Stereo Sound?

Thank you.

**Comment by mekya** [comment]
Hi @Mohit-3196,

Thank you for this question.  You can test the improvement through codepen. Please visit this discussion to show the samples
https://github.com/orgs/ant-media/discussions/5535

There is no need to set/enable anything, it comes with built-in feature and automatically enabled.

Cheers
Oguz
